app/Http/Requests/EditRoom.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class EditRoom extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|unique:rooms,name,'.$this->room->id
```

Comment:
Just like links, do we want room names to be unique? Two orgs may have the need for the same room name. As it stands, if they try to create a room with the same name, it will produce an error. Orgs should be unique... but not the rooms right?

Reply:
What use-case would that be useful with? i see that being more confusing for the users when you say hey join this room and there are 5 duplicate names